### ISSUE TITLE
Neurite dystrophy

### ISSUE THREAD

**Issue by sarahstofel** [obophenotype/human-phenotype-ontology#7052]
**Preferred term label:**
Neurite dystrophy

**Synonyms**
Dystrophic neurite

**Definition (free text, please give PubMed ID)**

Degenerated neurite that may result from genetic abnormalities or inadequate or faulty metabolism.
PMID:30899091

**Parent term (use hpo.jax.org/app)**

Morphological central nervous system abnormality

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: 104300

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0003-2307-1226

**Comment by pnrobinson** [comment]
added, thanks @sarahstofel
